lib/jsonapi/utils/support/pagination.rb

```diff
@@ -139,6 +139,16 @@ def count_records(records, options)
             records.except(:group, :order).count("DISTINCT #{records.table.name}.id")
           end
         end
+
+        def count_pages(records, options)
+          record_count = count_records(records, options)
+
+          case JSONAPI.configuration.default_paginator
```

Comment:
It would be great to add support to `:offset` paginators as well:

```ruby
def count_pages(record_count)
  return 0 unless record_count.to_i > 0
  size =
    case JSONAPI.configuration.default_paginator
    when :paged  then page_params['size']
    when :offset then page_params['limit']
    end
  size = size.to_i.nonzero? || JSONAPI.configuration.default_page_size
  (record_count.to_f / size).ceil
end
```